src/I2cAnalyzer.cpp

```diff
@@ -282,7 +282,8 @@ void I2cAnalyzer::AdvanceToStartBit()
                 break;
         }
     }
-    mResults->AddMarker( mSda->GetSampleNumber(), AnalyzerResults::Start, mSettings->mSdaChannel );
+
+    RecordStartStopBit();
```

Comment:
@Marcus10110 Is it okay to call:

```c++
    mResults->CommitPacketAndStartNewPacket();
    mResults->CommitResults();
```

at the very beginning? I wasn't sure what the semantics were.

Reply:
yup, CommitPacketAndStartNewPacket doesn't do anything yet anyway :) CommitResults does not do anything if the frame count is 0.